css/_container.scss

```diff
@@ -0,0 +1,283 @@
+@import url("https://fonts.googleapis.com/css2?family=Chivo:wght@300;700&display=swap");
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: $backgroundColor;
+  font-family: "Chivo", sans-serif;
+  height: 100vh;
+}
+
+main::before {
+  content: "";
+  background-image: url("/assets/mobile/image-host.jpg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  bottom: 0px;
+  left: 0px;
+  opacity: 0.1;
+  z-index: -1;
+}
+main {
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  align-items: center;
+  .logo {
+    margin: 62px 120.15px 57px 120px;
+  }
+  h1 {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 26px;
+    line-height: 38px;
+    text-align: center;
+    text-transform: uppercase;
+    color: $h1Color;
+    span {
+      color: $white;
+    }
+  }
+  p {
+    margin-top: 16px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 25px;
+    text-align: center;
+    color: $pColor;
+    margin-left: 24px;
+    margin-right: 24px;
+    opacity: 0.9;
+  }
+  .images {
+    display: flex;
+    justify-content: space-between;
+    margin: 33px 30px 38px 30px;
+    width: 315px;
+    opacity: 0.5;
+    .spotify {
+      width: 56px;
+      height: 17px;
+    }
+    .apple {
+      width: 45px;
+      height: 17px;
+    }
+    .google {
+      width: 73px;
+      height: 11px;
+    }
+    .pocket {
+      width: 77px;
+      height: 15px;
+    }
+  }
+  form {
+    #error {
+      font-style: normal;
+      font-size: 12px;
+      line-height: 14px;
+      margin-left: 107px;
+      color: #fb3e3e;
+      font-weight: 700;
+      margin-top: -8px;
+    }
+    input {
+      background: #2c344b;
+      border-radius: 28px;
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 28px;
+      color: #ffffff;
+      mix-blend-mode: normal;
+      border: none;
+      height: 46px;
+      width: 327px;
+      margin: 0px 24px 16px 24px;
+    }
+    input[type="text"]::placeholder {
+      color: $placeholderColor;
+      opacity: 0.8;
+    }
+    input[type="text"] {
+      font-weight: bold;
+      font-size: 14px;
+      line-height: 28px;
+      color: $white;
+      mix-blend-mode: normal;
+      padding-left: 32px;
+    }
+    input[type="submit"] {
+      background-color: $inputSubmitColor;
+      cursor: pointer;
+      color: #121725;
+      font-weight: bold;
+      font-size: 14px;
+      line-height: 28px;
+    }
+    input[type="submit"]:hover {
+      background-color: $hover;
+    }
+  }
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
Lose nestovanje i ovde. Media query za svaku klasu pisemo nestovano tamo gde prvi put deklarisemo klasu/element.

`
.myClass {
    ....
    @media only screen and (min-width: 768px) {
        ....
    }
}
`

Tako ces imati samo jednom dekleraciju elementa i sav responsive kod za nju odmah ispod - nestovano! 
Ovo je vazno da se ispravi jer si to imao i na proslom PR-u.